addons/im_livechat/models/discuss_channel.py

```diff
@@ -179,10 +179,11 @@ def _get_channel_history(self):
         for message in (self.message_ids - self.message_ids.sudo()._filter_empty()).sorted("id"):
             if message.author_id == chatbot_op and not last_msg_from_chatbot:
                 parts.append(Markup("<br/>"))
-            if message.author_id == chatbot_op:
-                parts.append(Markup("<strong>%s</strong><br/>") % html2plaintext(message.body))
-            else:
-                parts.append(Markup("%s<br/>") % html2plaintext(message.body))
+            if not tools.is_html_empty(message.body):
+                if message.author_id == chatbot_op:
+                    parts.append(Markup("<strong>%s</strong><br/>") % html2plaintext(message.body))
+                else:
+                    parts.append(Markup("%s<br/>") % html2plaintext(message.body))
             last_msg_from_chatbot = message.author_id == chatbot_op
```

Comment:
`last_msg_from_chatbot` should probably be updated only if the last shown message was from chatbot. Could you also update the test to take this case into account?

Reply:
There are other issues with this method aswell. For example 
<img width="237" height="168" alt="image" src="https://github.com/user-attachments/assets/8c94168c-47ce-4a13-84e5-4d898262f4df" />

As you can see Visitor's message is being displayed in bold here. Rather than human operator's. So, I was thinking about handling these issues in a separate PR rather than stretching this one. What do you think?